integreat_cms/cms/fixtures/test_data.json

```diff
@@ -508,6 +525,23 @@
       "last_updated": "2022-09-14T15:07:49.750Z"
     }
   },
+  {
+    "model": "cms.languagetreenode",
+    "pk": 10,
+    "fields": {
+      "lft": 3,
+      "rgt": 4,
+      "tree_id": 1,
```

Comment:
Adding this kind of works, but leaves inconsistencies in the database. Inserting an element in an MPTT structure always changes at least the `rgt` value of the root node - each `lft` and `rgt` value may only exist once in each tree:

![mptt](https://user-images.githubusercontent.com/16021269/206284584-9a3edfb8-7166-421a-b417-9abae0e5eb7f.png)

But you don't have to understand mptt in order to create correct tree structures - our tree library will do this for us. Just create the language via the GUI and then export the table via:
```
export INTEGREAT_CMS_SECRET_KEY="dummy"
export INTEGREAT_CMS_DEBUG=1
pipenv run integreat-cms-cli dumpdata cms.languagetreenode --settings=integreat_cms.core.docker_settings
```
And then overwrite the existing test tree nodes with the output.